arch/wasm/bits/limits.h

```diff
@@ -0,0 +1,17 @@
+#if defined(_POSIX_SOURCE) || defined(_POSIX_C_SOURCE) \
+ || defined(_XOPEN_SOURCE) || defined(_GNU_SOURCE) || defined(_BSD_SOURCE)
+/*
+ * How big should our pages be?  There is *no requirement at all* that our
+ * emulated kernel page size matches the underlying page size used by the
+ * Wasm memory builtins!  Wasm pages are 64KiB, which is pretty huge.
+ * malloc() switches to using mmap around the 128KiB mark, which means a
+ * 129KiB allocation would be wasting 33% of the RAM if we did 64KiB mmap
+ * allocations.  At 32KiB page size, we have <20% RAM
+ * waste which seems a bit more reasonable.  All a bit arbitrary.
+ */
+#define PAGE_SIZE 32768
```

Comment:
(Random drive-by review comment...)

This property of page sizes may not always hold; it is possible that wasm could add [mmap-like features in the future](https://github.com/WebAssembly/design/blob/master/FutureFeatures.md#finer-grained-control-over-memory), in which case PAGE_SIZE should reflect the granularity of those operations.

I realize that you don't want to mix in bigger changes in this PR, but if malloc's heuristics are suboptimal for 64KiB page sizes, it would affect other architectures as well, including some arm64 platforms, so ideally the solution should be to fix malloc's heuristics.